infrastructure/critical/modules/rds/security_groups.tf

```diff
@@ -0,0 +1,30 @@
+resource "aws_security_group" "database_sg" {
+  description = "controls direct access to application instances"
+  vpc_id      = var.vpc_id
+  name        = "${var.database_name}_database_sg"
+
+  ingress {
+    protocol  = "tcp"
+    from_port = 3306
+    to_port   = 3306
+
+    cidr_blocks = [
+      var.admin_cidr_ingress,
+    ]
+  }
+
+  ingress {
+    from_port = 3306
+    to_port   = 3306
+    protocol  = "tcp"
+
+    security_groups = var.db_access_security_group
+  }
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+}
```

Comment:
@alexwlchan actually on this - I might not change anything, the reason being that will update the state, and thus the RDS instance, which, honestly, I don't want to do given the sensitive nature of the data.

Reply:
Take it back, it doesn't force and remaking of resources.